app/controllers/photos_controller.rb

```diff
@@ -19,8 +19,9 @@ def index
   def new
     @type = params[:type]
     @id = params[:id]
-
     @photo = Photo.new
+    planting = Planting.find @id
```

Comment:
It might not be a planting. It might be a garden

There's some code later in the same file, that turns the item type and id into the model itself
`
    item_class = Growstuff::Constants::PhotoModels.get_item(params[:type])
    item = item_class.find_by!(id: params[:id], owner_id: current_member.id)
`
That'll look up the actual model, whatever type it is.